backend/tests/core/TestHelperExtensions.cs

```diff
@@ -22,17 +25,26 @@ public static class TestHelperExtensions
         /// <param name="helper"></param>
         /// <param name="args"></param>
         /// <typeparam name="T"></typeparam>
-        public static void MockConstructorArguments<T>(this TestHelper helper, params object[] args)
+        /// <returns></returns>
```

Comment:
returns a Dictionary?

Reply:
Good catch!  Just realized while this will work great for most use-cases, it is possible to have multiple arguments of the same type.  I do like dictionaries because they're easy to grab the correct key/value.  I'll switch to an array of `KeyValuePair<,>`